From f95d68c06c54cc741a67dbc5cdc1088fab58c40c Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Thu, 13 May 2010 15:22:40 +0800
Subject: [PATCH 07/69] ENGR00123440 Correct the firmware install path

Align with the WIFI firmware loading in linux-atheros-wifi.git
Align with the VPU firmware loading in linux-lib.git

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 firmware/ath6k/Android.mk | 11 ++++++-----
 firmware/vpu/Android.mk   |  4 +---
 2 files changed, 7 insertions(+), 8 deletions(-)

diff --git a/firmware/ath6k/Android.mk b/firmware/ath6k/Android.mk
index 7341e72..cb7e397 100644
--- a/firmware/ath6k/Android.mk
+++ b/firmware/ath6k/Android.mk
@@ -4,12 +4,13 @@ ifeq ($(BOARD_HAVE_WIFI), true)
 
 LOCAL_PATH := $(call my-dir)
 AR_FW_PATH := AR6102
+INSTALL_PATH := $(TARGET_OUT)/lib/firmware/ath6k/$(AR_FW_PATH)
 
 include $(CLEAR_VARS)
 LOCAL_MODULE := athwlan.bin.z77
 LOCAL_MODULE_TAGS := user
 LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(TARGET_OUT)/wifi
+LOCAL_MODULE_PATH := $(INSTALL_PATH)
 LOCAL_SRC_FILES := $(AR_FW_PATH)/$(LOCAL_MODULE)
 include $(BUILD_PREBUILT)
 
@@ -17,7 +18,7 @@ include $(CLEAR_VARS)
 LOCAL_MODULE := data.patch.hw2_0.bin
 LOCAL_MODULE_TAGS := user
 LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(TARGET_OUT)/wifi
+LOCAL_MODULE_PATH := $(INSTALL_PATH)
 LOCAL_SRC_FILES := $(AR_FW_PATH)/$(LOCAL_MODULE)
 include $(BUILD_PREBUILT)
 
@@ -25,7 +26,7 @@ include $(CLEAR_VARS)
 LOCAL_MODULE := eeprom.data
 LOCAL_MODULE_TAGS := user
 LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(TARGET_OUT)/wifi
+LOCAL_MODULE_PATH := $(INSTALL_PATH)
 LOCAL_SRC_FILES := $(AR_FW_PATH)/$(LOCAL_MODULE)
 include $(BUILD_PREBUILT)
 
@@ -33,7 +34,7 @@ include $(CLEAR_VARS)
 LOCAL_MODULE := eeprom.bin
 LOCAL_MODULE_TAGS := user
 LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(TARGET_OUT)/wifi
+LOCAL_MODULE_PATH := $(INSTALL_PATH)
 LOCAL_SRC_FILES := $(AR_FW_PATH)/$(LOCAL_MODULE)
 include $(BUILD_PREBUILT)
 
@@ -41,7 +42,7 @@ include $(CLEAR_VARS)
 LOCAL_MODULE := calData_ar6102_15dBm.bin
 LOCAL_MODULE_TAGS := user
 LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(TARGET_OUT)/wifi
+LOCAL_MODULE_PATH := $(INSTALL_PATH)
 LOCAL_SRC_FILES := $(AR_FW_PATH)/$(LOCAL_MODULE)
 include $(BUILD_PREBUILT)
 
diff --git a/firmware/vpu/Android.mk b/firmware/vpu/Android.mk
index 87fa5e0..5d10d01 100644
--- a/firmware/vpu/Android.mk
+++ b/firmware/vpu/Android.mk
@@ -3,9 +3,7 @@ ifeq ($(BOARD_HAVE_VPU),true)
 
 LOCAL_PATH := $(call my-dir)
 
-vpu_etc_dir := $(TARGET_OUT)/etc/vpu
-vpu_fw_dir := vpu_fw
-vpu_fw_target := $(vpu_etc_dir)/$(vpu_fw_dir)
+vpu_fw_target := $(TARGET_OUT)/lib/firmware/vpu
 
 soc := $(shell echo "$(BOARD_SOC_TYPE)" | tr [A-Z] [a-z])
 vpu_fw_file := vpu_fw_$(soc).bin
-- 
1.8.0

